itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -2851,6 +2854,8 @@ public void testViewsReplication() throws IOException {
     verifyRun("SELECT * from " + replDbName + ".virtual_view", empty, driverMirror);
     //verifyRun("SELECT a from " + replDbName + ".mat_view", ptn_data_1, driverMirror);
 
+    verifySetup("SELECT * from " + replDbName + ".virtual_view_with_partition", empty, driver);
```

Comment:
Shouldn't the same issue be there in incremental case also? If so, I think we should add a test for incremental also and modify the bug synopsis by removing 'bootstrap' word in it.

Reply:
Also, can we have one non-empty table also covered?